src/animations/CruisePanelsAnimation.tsx

```diff
@@ -1,64 +1,38 @@
-import React, { useRef, useState } from 'react';
-import { useFrame } from '@react-three/fiber';
-import { Group } from 'three';
-import useAnimation from './use-animation';
-import AnimationName from './types/animation-name';
-import { Earth } from '../artifacts/Earth';
-import filledVector from '../common/utils/filled-vector';
-import { CruiseOrbiter } from '../artifacts/CruiseOrbiter';
-
-const earthScale = filledVector(5);
-const orbiterScale = filledVector(0.75);
+import React, { useEffect } from 'react';
+import useExplode from '../common/explode/use-explode.ts';
+import useAnimation from './use-animation.ts';
+import AnimationName from './types/animation-name.ts';
 
 /**
  * Cruise Solar Panel Animation
  */
-const CruisePanelsAnimation: React.FC<JSX.IntrinsicElements['group']> = ({
-  ...props
-}) => {
+const CruisePanelsAnimation: React.FC = () => {
   const { isAnimationActive, stopAnimation } = useAnimation();
-  const [elapsed, setElapsed] = useState(0);
-  const earthRef = useRef<Group>(null);
-  const orbiterRef = useRef<Group>(null);
-  // const isActive = useMemo(() => {
-  //   return isAnimationActive(AnimationName.CRUISE_PANELS);
-  // }, [isAnimationActive]);
+  const { isExploded, toggleExploded, isAtRest } = useExplode();
 
-  useFrame((_, delta) => {
-    if (earthRef.current == null || orbiterRef.current == null) {
+  useEffect(() => {
+    if (!isAnimationActive(AnimationName.CRUISE_PANELS)) {
       return;
     }
 
-    if (!isAnimationActive(AnimationName.CRUISE_PANELS)) {
-      return;
+    if (isExploded) {
+      toggleExploded();
     }
 
-    setElapsed(elapsed + delta);
+    let timeout: NodeJS.Timeout;
```

Comment:
You can remove this and just declare on line 25, `const timeout = setTimeout(...)`